bindings/Fuse.res

```diff
@@ -0,0 +1,29 @@
+module Options = {
+  @bs.deriving(abstract)
+  type t = {
+    @bs.optional
+    shouldSort: bool,
+    @bs.optional
+    includeScore: bool,
+    @bs.optional
+    threshold: float,
+    @bs.optional
+    location: int,
+    @bs.optional
+    distance: int,
+    @bs.optional
+    minMatchCharLength: int,
+    @bs.optional
+    keys: array<string>,
```

Comment:
FYI, `bs.` isn not needed any more

Reply:
old habits 😅 ... wanted to remove all occurrences of `bs.` after we upgraded to `8.4`